pyaedt/edb_core/components.py

```diff
@@ -1,4 +1,4 @@
-"""This module contain the `Components` class.
+"""This module contains the `Components` class."""
```

Comment:
```suggestion
"""This module contains the `Components` class.
```
This is causing the CI to fail.

Reply:
Looks like you fixed that... consider this resolved.